gtfu/src/main/java/gtfu/Util.java

```diff
@@ -846,4 +846,32 @@ public static String getGTFSURL(AgencyData[] list, String id) {
         return collections;
     }
 
-}
\ No newline at end of file
+    public static Map<String, String> hashMapFromTextFile(String path, String delimiter) {
+
+        Map<String, String> map = new HashMap<String, String>();
+        BufferedReader br = null;
+
+        try {
+            File file = new File(path);
+            br = new BufferedReader(new FileReader(file));
+            String line = null;
+
+            while ((line = br.readLine()) != null) {
+
+                String[] parts = line.split(delimiter);
+                String name = parts[0].trim();
+                String recipients = parts[1].trim();
+
+                if (!name.equals("") && !recipients.equals("")){
+                    map.put(name, recipients);
+                }
+            }
+        }
+        catch (Exception e) {
+            System.out.println("** error: no such file " + path);
+            System.exit(1);
+        }
+
+        return map;
+    }
+}
```

Comment:
I don't like this approach. It's very limited. Instead, please consider:

- recipients.json:
```
    {
      'error_report': [
        'scott.owades@dot.ca.gov', 
        'kay.neuenhofen@dot.ca.gov'
      ],
      'graas_report': [
        'scott.owades@dot.ca.gov', 
        'lauren.gilbert@rebelgroup.com, 
        'ali.attari@rebelgroup.com', 
        'kay.neuenhofen@dot.ca.gov'
      ]
  }
```

- Util.java:
  ```
  public static Map<String, Object> parseJSONasMap(String path) {
        try {
            return new ObjectMapper().readValue(path, HashMap.class);;
        } catch (IOException e) {
            throw new Fail(e);
        }
  }
```